src/components/details-view/details-view.js

```diff
@@ -198,6 +199,12 @@ export default class DetailsView extends Component {
 
     let historyState = router.history.location.state;
 
+    let countFilters = (params) => {
```

Comment:
Now that I'm looking at this again, I don't think this needs to be a function at all since it's called immediately below, and it's always with `searchParams`.

We could probably just move the logic into a single assignment:

```js
let filterCount = [searchParams.q]
  .concat(Object.values(searchParams.filter))
  .filter(Boolean).length;
```